test-network/README.md

```diff
@@ -2,6 +2,13 @@
 
 You can use the `./network.sh` script to stand up a simple Fabric test network. The test network has two peer organizations with one peer each and a single node raft ordering service. You can also use the `./network.sh` script to create channels and deploy chaincode. For more information, see [Using the Fabric test network](https://hyperledger-fabric.readthedocs.io/en/latest/test_network.html). The test network is being introduced in Fabric v2.0 as the long term replacement for the `first-network` sample.
 
+If you are planning to run the test network with consesnsus type BFT then please pass `-bft` flag as input to the `network.sh` script when creating the channel.
+That is to create a channel use:
+
+```bash
+./network.sh createChannel -bft
+```
+
```

Comment:
Need to say the same thing about
`./network.sh up -bft`

So that the four orderer nodes start up.